src/types/integer.rs

```diff
@@ -9,7 +9,7 @@ use std::mem;
 use std::ops::{Add, Neg};
 
 /// Represents an unsigned integer of any size.
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
🗺️ Starting in v1.80, Clippy correctly identified that a lot of our manual implementations could be derived.